.github/workflows/mobilecoin-workflow-dev-delete.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+#
+# MobileCoin Core projects - Reusable Workflow - Delete development namespace and associated components.
+
+name: mobilecoin-workflow-dev-delete
```

Comment:
Basic question: Why do we need/have a `workflow_dispatch` and `workflow_call` variant of each workflow?